aten/src/ATen/native/quantized/ConvUtils.cpp

```diff
@@ -0,0 +1,47 @@
+#include <ATen/native/quantized/ConvUtils.h>
+
+#if defined(USE_CUDA) || defined(USE_PYTORCH_QNNPACK)
+namespace at::native::quantized {
+template <>
+at::SmallVector<int64_t, 4> MakeConvOutputShape<2>(
```

Comment:
are these related to quantized convs specifically? if not, maybe worth moving them up in the namespace?

Reply:
They are only used in quantized conv, though there is nothing that screams quantized here. But I'll likely have to move them to anonymous namespace and force inlining as Windows linking quirks drive me crazy.

Reply:
The reason I'm asking is that in more frontend context it was decided that this formula should not be available explicitly and that meta tensors + module/function call should be done for computing this :) So I thought that this might weigh on the other side and a simple user-facing formula can be provided in some torch.nn.utils :)

Reply:
Makes sense, and I guess it could.